exercises/sgf-parsing/description.md

```diff
@@ -53,6 +53,19 @@ A key can have multiple values associated with it. For example:
 
 Here `AB` (add black) is used to add three black stones to the board.
 
+According the the SGF specification `\` is the escape character. When `\` is
+encountered during parsing it indicates that the following character should be
+printed as is. The exception to this is whitespace, as is common with many
+encoding schemas the character sequences `"\n"`, `"\r"` and `"\t"` are the
+escaped forms of the newline, carriage return and space characters
+respectively. When parsing you can expect some characters to always be escaped
+including parentheses, square brackets, semicolons and colons.
```

Comment:
Maybe shorten this a bit.

```suggestion
printed as is. The exception to this is whitespace, with newline, carriage return and 
tabs encoded as `"\n"`, `"\r"` and `"\t"` respectively. When parsing you can expect some characters to _always_ be escaped, including parentheses, square brackets, semicolons 
and colons.
```

There was also a small mistake, where is said the `\t` encoded spaces.